### PR TITLE
Remove operator-sdk leftovers

### PR DESCRIPTION
**Pull request by honza**
* Documentation
* Travis config
* Hack script pruning

### CODE REVIEW

File: hack/markdownlint.sh [hunk 1]
```diff
@@ -8,8 +8,7 @@ CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 if [ "${IS_CONTAINER}" != "false" ]; then
   TOP_DIR="${1:-.}"
   find "${TOP_DIR}" \
-       \( -path ./vendor -prune -o \
-       -path ./tools/operator-sdk -prune \) \
+       \( -path ./vendor -prune -o \) \
```
Comment (dhellmann) [inline hack/markdownlint.sh:11]: I don't think we need the `-o` there?
```suggestion
       \( -path ./vendor -prune \) \
```

File: hack/shellcheck.sh [hunk 2]
```diff
@@ -8,8 +8,7 @@ CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 if [ "${IS_CONTAINER}" != "false" ]; then
   TOP_DIR="${1:-.}"
   find "${TOP_DIR}" \
-       \( -path ./vendor -prune -o \
-       -path ./tools/operator-sdk -prune \) \
+       \( -path ./vendor -prune -o \) \
```
Comment (dhellmann) [inline hack/shellcheck.sh:11]: Same
```suggestion
       \( -path ./vendor -prune \) \
```

File: Makefile [hunk 3]
```diff
@@ -132,6 +132,10 @@ run: generate fmt vet manifests ## Run against the configured Kubernetes cluster
 demo: generate fmt vet manifests ## Run in demo mode
 	go run -ldflags $(LDFLAGS) ./main.go -namespace=$(RUN_NAMESPACE) -dev -demo-mode
 
+.PHONY: run-test-mode
+run: generate fmt vet manifests ## Run against the configured Kubernetes cluster in ~/.kube/config
```
Comment (zaneb) [inline Makefile:136]: `run-test-mode:`

Comment (honza) [reply Makefile:136]: :facepalm:

**Comment by furkatgofurov7** [comment]
CI glitch. Re-triggering it again. 
/test-integration

**Pull request merged**